configuration/raspi-config.md

```diff
@@ -56,42 +54,69 @@ Generally speaking, `raspi-config` aims to provide the functionality to make the
  
-<a name="network-options"></a>
-### Network Options
-
-From this submenu you can set the host name, your wireless LAN SSID, and pre-shared key, or enable/disable predictable network interface names.
-
 <a name="hostname"></a>
 #### Hostname
 
 Set the visible name for this Pi on a network.
 
 <a name="boot-options"></a>
-### Boot Options
+#### Boot / Auto login
 
 From here you can change what happens when your Pi boots. Use this option to change your boot preference to command line or desktop. You can choose whether boot-up waits for the network to be available, and whether the Plymouth splash screen is displayed at boot-up.
 
-<a name="localisation-options"></a>
-### Localisation Options
+#### Network at Boot
 
-The localisation submenu gives you these options to choose from: keyboard layout, time zone, locale, and wireless LAN country code.
+Use this option to wait for a network connection before letting boot proceed.
 
-#### Change locale
-Select a locale, for example `en_GB.UTF-8 UTF-8`.
+#### Splash Screen
 
-#### Change time zone
-Select your local time zone, starting with the region, e.g. Europe, then selecting a city, e.g. London. Type a letter to skip down the list to that point in the alphabet.
+Enable or disable the splash screen displayed at boot time
 
-#### Change keyboard layout
-This option opens another menu which allows you to select your keyboard layout. It will take a long time to display while it reads all the keyboard types. Changes usually take effect immediately, but may require a reboot.
+#### Power LED
 
-#### Change wireless country
-This option sets the country code for your wireless network.
+If the model of Pi permits it, you can change the behaviour of the power LED using this option.
+
+### Display Options
+
+<a name="resolution"></a>
+#### Resolution
+
+Define the default HDMI/DVI video resolution to use when the system boots without a TV or monitor being connected. This can have an effect on RealVNC if the VNC option is enabled.
+
+<a name="overscan"></a>
+#### Overscan
+
+Old TV sets had a significant variation in the size of the picture they produced; some had cabinets that overlapped the screen. TV pictures were therefore given a black border so that none of the picture was lost; this is called overscan. Modern TVs and monitors don't need the border, and the signal doesn't allow for it. If the initial text shown on the screen disappears off the edge, you need to enable overscan to bring the border back.
+
+Any changes will take effect after a reboot. You can have greater control over the settings by editing [config.txt](config-txt/README.md).
+
+On some displays, particularly monitors, disabling overscan will make the picture fill the whole screen and correct the resolution. For other displays, it may be necessary to leave overscan enabled and adjust its values.
+
+<a name="pixel-doubling"></a>
+#### Pixel Doubling
+
+Enable/disable 2x2 pixel mapping.
+
+#### Screen Blanking
```

Comment:
My version of `raspi-config` has a "Composite Video" menu option between the "Pixel Doubling" and "Screen Blanking" options?

Reply:
Doesn't appear on Pi4 I have here...